seeq/addons/correlation/_seeq_worksheet_writer.py

```diff
@@ -214,7 +215,17 @@ def worksheet_corrs_and_time_shifts(signal_pairs_ids: list, workbook_id: str,
                             condition_id,
                             overwrite,
                             api_client)
-    return "/".join([spy.client.host.replace('/api', ""), 'workbook', workbook_id, 'worksheet', existing_worksheet.id])
+
+    seeq_url_ = get_seeq_url()
+    
+    if seeq_url_ is None:
+        try:
+            with open('.urlfile/seeq_server_url.pkl', 'rb') as file_:
+                seeq_url_ = pickle.load(file_)
+        except:
```

Comment:
This is a broad exception. You should only catch the exception you mean to catch `except FileNotFoundError as error:` then you could you `error` to log the actual exception (e.g. path of the file/directory that was trying to find)